draft-ietf-quic-transport.md

```diff
@@ -2057,11 +2057,17 @@ linked by any other entity.
 At any time, endpoints MAY change the Destination Connection ID they send to a
 value that has not been used on another path.
 
-An endpoint MUST use a new connection ID if it initiates connection migration.
-Using a new connection ID eliminates the use of the connection ID for linking
-activity from the same connection on different networks.  Header protection
-ensures that packet numbers cannot be used to correlate activity.  This does not
-prevent other properties of packets, such as timing and size, from being used to
+An endpoint MUST use a new connection ID if it initiates connection migration as
+described in {{initiating-migration}}.  An endpoint MUST use a new
+connection ID in response to a connection migration if the packet that initiates
+migration uses a different connection ID to packets received on any previous
+path.
```

Comment:
I wonder if using the term "initiate a connection migration" is correct, as to me it seems to imply what [section 9.2](https://quicwg.org/base-drafts/draft-ietf-quic-transport.html#rfc.section.9.2) talks, but not [path probing (section 9.1)](https://quicwg.org/base-drafts/draft-ietf-quic-transport.html#rfc.section.9.1).

Shouldn't we better state that, an endpoint MUST use a new connection ID when responding to a packet on a new path, using a new connection ID?

Reply:
Is it feasible to remember all past CID's?

Reply:
Maybe, but you are right, this could be better.  I just don't know how to word that without turning the sentence into an essay.  Suggestions sought.

Reply:
The server must have a short list if issued CID's. It suffice that the CID is in that list, and that the client does not reuse old CID's - as is already the case. So something like 

```
An endpoint MUST use a new
connection ID in response to a connection migration that uses a new connection ID from the list of issued connection ID's.```